contentcuration/contentcuration/static/js/edit_channel/constants/index.js

```diff
@@ -9,7 +10,7 @@ const Statuses = require('./Statuses');
 module.exports = {
   ContentKinds,
   FormatPresets,
-  Languages,
+  Languages: sortBy(Languages, 'readable_name'),
```

Comment:
Yess! Super small request, but could you use `native_name` instead? It's a bit more flexible when Studio is displayed in other languages